src/schema/v2/fair.ts

```diff
@@ -381,7 +381,13 @@ export const FairType = new GraphQLObjectType<any, ResolverContext>({
             for (const fairExhibitor of fairExhibitors) {
               const names = fairExhibitor.name.split(" ")
               const firstName = names[0]
-              const letter = firstName.charAt(0).toUpperCase()
+              let letter = firstName.charAt(0).toUpperCase()
+
+              // Numeric or special symbol
+              if (/[^A-Z]/i.test(letter)) {
```

Comment:
**nitpick (non-blocking):** This regular expression will also catch exhibitors whose names start with accented characters like `Á` or `Ö`. I don't know if there are any galleries in our system that start with accented characters, so I am marking this as non-blocking. (I couldn't find any in the ongoing fairs.)

Reply:
Yes, I also thought about this case, but I also didn't find such names in the system. To solve this problem, we can use `deburr` from [lodash](https://lodash.com/docs/4.17.15#deburr) and move exhibitors whose names start with accented characters (like, `Á`) into a group with the letter `A` ([on the frontend](https://github.com/artsy/force/blob/179755b196356b5b93681f288fe9273cd3dff8e4/src/v2/Apps/Fair/Components/ExhibitorsLetterNav.tsx#L11), we can only choose from the values `A-Z`, `#`)

Reply:
That's a wonderful solution! It makes sense that any exhibitor starting with `Á`, `Å`, etc. would be grouped with `A`.

Reply:
I think using `deburr` is a good idea; I also don't immediately see any partners whose names start with special characters but since it doesn't add a ton of complexity I think it's worth doing